autoload/instrument_list.gd

```diff
@@ -0,0 +1,43 @@
+extends Node
+
+var instruments: Dictionary = {}
+
+# TODO : change this to PacketStringArray later on. currently for loops don't support it.
```

Comment:
You can use Array[String], which is type safe but doesn't have the restrictions of a PackedArray